assets/materials/1y.mat

```diff
@@ -0,0 +1,8 @@
+{
+    "shader" : "engine:genericMeshMaterial",
+    "params" : {
+        "diffuse" : "yellowGooey",
```

Comment:
Seeing as how you've named all the other child textures as `1r2b` and the materials follow the same pattern, I'd recommend renaming the texture to `1y` here (and `1r` & `1b` likewise)